docs/concepts/actions.md

```diff
@@ -0,0 +1,64 @@
+# Actions
+
+Actions are _the_ way to change the state. Actions have this signature: `payload => (state, actions) => newState`. Actions are passed to the `app` as the second parameter ([app](tbd)).
+
+## Usage
+
+Actions get the payload passed as parameter of the first function. This can return an `object` (the new state) or a function. When returning an `object` it'll be (shallow) merged with the state. Otherwise it'll be called and passed the `state` and `actions` which in turn returns a new state object that will be (shallow) merged.
+
+- `() => newState`
+- `payload => newState`
+- `payload => (state, actions) => newState`
```

Comment:
I don't really like using `code = like > this` unless it is for a single variable. It doesn't look very pretty.

Reply:
I like it... but what would you suggest?